Library/Homebrew/cmd/cleanup.rb

```diff
@@ -108,10 +108,7 @@ def cleanup_lockfiles
   end
 
   def rm_DS_Store
-    paths = %w[Cellar Frameworks Library bin etc include lib opt sbin share var].
-      map { |p| HOMEBREW_PREFIX/p }.select(&:exist?)
```

Comment:
Because we want to clean only the directories used by Homebrew, not the others. The default prefix is `/usr/local`, which is also used outside of Homebrew.